### PR TITLE
Adds new methods to article class

### PR DESCRIPTION
**Pull request by ashok-arora**

### CODE REVIEW

File: tests/stress_test.py [hunk 1]
```diff
@@ -15,11 +15,11 @@
     print('Testing Passed')
 
 except exceptions.ParseError:
-    print('Testing Failed: ParseError')
-    print('Report the error with tha way to reproduce it at vigilant.umbrella@gmail.com')
+    print('Testing Failed: ParseError @')
```
Comment (aniketsharma00411) [inline tests/stress_test.py:18]: Why have you added '@' at the end?

File: examples/article_info.py [hunk 2]
```diff
@@ -25,3 +25,9 @@
 print(first_step.title)
 print(first_step.description)
 print(first_step.picture)
+
+print(article.co_authors)
```
Comment (aniketsharma00411) [inline examples/article_info.py:29]: Shouldn't we keep all print statements of article data members together? After line 20.

File: examples/article_info_using_dictionary.py [hunk 3]
```diff
@@ -22,3 +22,9 @@
 print(first_step.title)
 print(first_step.description)
 print(first_step.picture)
+
+print(info['co_authors'])
```
Comment (aniketsharma00411) [inline examples/article_info_using_dictionary.py:26]: Shouldn't we keep all print statements of article data members together? After line 17.

File: wikihowunofficialapi/__init__.py [hunk 4]
```diff
@@ -91,8 +91,13 @@ def __init__(self, url='https://www.wikihow.com/Special:Randomizer', lazy=True):
         self._is_expert = None
         self._last_updated = None
         self._views = None
-        self._parsed = False
+        self._co_authors = None
+        self._references = None
+        self._summary = None
```
Comment (aniketsharma00411) [inline wikihowunofficialapi/__init__.py:96]: '_summary' is already defined at line 87.

File: wikihowunofficialapi/__init__.py [hunk 5]
```diff
@@ -163,6 +168,36 @@ def views(self):
...
+        return self._co_authors
+
+    @property
+    def references(self):
+        if not self._parsed:
+            self._parse()
+        return self._references
+
+    @property
+    def summary(self):
```
Comment (aniketsharma00411) [inline wikihowunofficialapi/__init__.py:184]: This is also already defined at line 126.

File: wikihowunofficialapi/__init__.py [hunk 6]
```diff
@@ -317,7 +410,12 @@ def get(self):
             'percent_helpful': self.percent_helpful,
             'is_expert': self.is_expert,
             'last_updated': self.last_updated,
-            'views': self.views
+            'views': self.views,
+            'co_authors': self.co_authors,
+            'references': self.references,
+            'summary': self.summary,
```
Comment (aniketsharma00411) [inline wikihowunofficialapi/__init__.py:416]: Also, already there at line 408.

File: wikihowunofficialapi/__init__.py [hunk 7]
```diff
@@ -289,6 +323,59 @@ def _parse_views(self, soup):
...
+            for reference in references_html:
+                count += 1
+        self._references = count
+
+    def _parse_summary(self, soup):
+        summary_html = soup.find('p', {'id': 'summary_text'})
+        if not summary_html:
+            return None
+        else:
+            self._summary = summary_html.text
```
Comment (aniketsharma00411) [inline wikihowunofficialapi/__init__.py:351]: At the end of the summary it prints "Did this summary help you?YesNo", which should not be included in it. Could you see what can be done?

**Review by aniketsharma00411 — CHANGES_REQUESTED** [review]

File: tests/stress_test.py (hunk 1 above)
Comment (ashok-arora) [reply tests/stress_test.py:18]: Sorry, I will remove that.

File: examples/article_info.py (hunk 2 above)
Comment (ashok-arora) [reply examples/article_info.py:29]: Okay, I will do that.

File: examples/article_info_using_dictionary.py (hunk 3 above)
Comment (ashok-arora) [reply examples/article_info_using_dictionary.py:26]: Okay, I will do that.

File: wikihowunofficialapi/__init__.py (hunk 7 above)
Comment (ashok-arora) [reply wikihowunofficialapi/__init__.py:351]: Okay, I will look into it!

**Review by aniketsharma00411 — APPROVED** [review]
Everything looks fine to me.

**Pull request merged**